### PR TITLE
Applications and Forms api integration in applicant app

### PR DESCRIPTION
**Pull request by devendra-tarento**
## Description

Added AxiosService, ApplicationService, FormService and integrates them in relevant page in. applicant page.

### Changes

List the changes done to fix a bug or introducing a new feature.

## How to test

Describe the steps required to test the changes proposed in the pull request.

Please consider using the closing keyword if the pull request is proposed to
fix an issue already created in the repository
(https://help.github.com/articles/closing-issues-using-keywords/)

### CODE REVIEW

**Comment by vShyave** [comment]
@devendra-tarento  hope you have raised this PR against this repo
Samagra-Development/affiliation-assessor-app

File: apps/wrapper/applicant-portal/.env
Comment (amit-s19) [inline apps/wrapper/applicant-portal/.env:1]: Please don't commit envs! Remove these asap.

File: apps/wrapper/applicant-portal/src/components/ApplicationCard.jsx
Comment (amit-s19) [inline apps/wrapper/applicant-portal/src/components/ApplicationCard.jsx:1]: Use conditional chaining; `props.application.form_name` -> `props?.application?.form_name`

File: apps/wrapper/applicant-portal/src/components/FormCard.jsx
Comment (amit-s19) [inline apps/wrapper/applicant-portal/src/components/FormCard.jsx:1]: Follow conditional chaining here too.

**Review by amit-s19 — APPROVED** [review]
Merging with comments

**Pull request merged**